### ISSUE TITLE
cleanup Physician per #806 to clarify that a physician is not a place, adding /usNPI identifiers

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#3420]
We have a longstanding issue (partially cleaned in #806) due to the choice of naming for the type Physician, which represents "a doctor's office" and has a variety of supertypes:

* [Thing](https://schema.org/Thing) > [Organization](https://schema.org/Organization) > [LocalBusiness](https://schema.org/LocalBusiness) > [MedicalBusiness](https://schema.org/MedicalBusiness) > [Physician](https://schema.org/Physician)
* [Thing](https://schema.org/Thing) > [Place](https://schema.org/Place) > [LocalBusiness](https://schema.org/LocalBusiness) > [MedicalBusiness](https://schema.org/MedicalBusiness) > [Physician](https://schema.org/Physician)
* [Thing](https://schema.org/Thing) > [Organization](https://schema.org/Organization) > [MedicalOrganization](https://schema.org/MedicalOrganization) > [Physician](https://schema.org/Physician)

The current poor typing as a LocalBusiness (via MedicalBusiness) brings in the subtype of Place. In #806 we decoupled MedicalOrganization from this hierarchy (although note that such organizations can have an address). We should continue this cleanup by removing the assertion that a Physician is a MedicalBusiness, leaving it typed just as a MedicalOrganization. For association with places we can draw attention to the [address](https://schema.org/address) and [hospitalAffiliation](https://schema.org/hospitalAffiliation) properties, with potential to add /practicesAt for more specificity and to cover clinics etc.

# Changes

1.) remove: Physician subClassOf LocalBusiness.

2.) Reword definition. Instead of "A physician's office", something like:
 
> An individual physician, considered as a [[MedicalOrganization]]. For their official address use [[address], for affiliations to hospitals use [[hospitalAffiliation]]. The (*todo*) [[practicesAt]] property can be used to indicate [[MedicalOrganization]] hospitals, clinkics, pharmacies etc. where this physician practices. If there is a need to describe the physician as a [[Person]], the [[founder]] property can be used.

3.) add /usNPI, a /Property whose value is /Text (using /Number doesn't add much here); domainIncludes /Physician.

> A US [NPI](https://en.wikipedia.org/wiki/National_Provider_Identifier) (National Provider Number) is a unique 10-digit identification number issued to health care providers in the US.

4.) Add /practicesAt, a /Property whose value is a /MedicalOrganization; domainIncludes /Physician.

Definition:
> Indicates a medical organization where this physician practices.

**Comment by jvandriel** [comment]
Maybe a crazy thought but might not `Physician` be better as a subtype of `Occupation`?

Than we could express: 
- `Person > hasOccupation > Physician` and
- `Physician > occupationLocation > AdministrativeArea`

**Comment by jvandriel** [comment]
Oh, and as for animals, isn't it about time schema.org/Animal was added to the vocabulary (and maybe also schema.org/Plant)?

Even if they're just simple classes, with not that many properties, it would make things a lot easier just having those basic classes.

**Comment by LeezaRodriguez** [comment]
I am happy to see that the chickens have come home to roost about the `Physician` Type .  I admit it took longer than I thought. Before this gets cleaned up, this is what **you need to know**:

## NPI numbers:  Two types (Individual Person, Organization)

CMS assigns two types of NPI Numbers: https://github.com/schemaorg/schemaorg/issues/806#issuecomment-164163649
- Individual person  (physician, person who practices medicine)
- Organization (medical practice, Facility, supplier, hospital, etc)

In order to assign the NPI numbers properly, we must disambiguate the Physician person from the Physician organization:

>In the USA, the US government disambiguates between the physician person and the physician office. The physician person and the physician organization each apply for separate NPI numbers. You do not exist as an individual provider person or a medical organization if you do not have an NPI number. With insurance claims, carriers require the NPI numbers of both the individual physician and the physician organization. Also, an individual physician can practice medicine with several different physician organizations. Malpractice policies, claims, and associated data bank history are mainly associated with individual physicians. Individual physicians must have state licenses to practice medicine. Not all medical organizations will receive a state license , as often times the burden rests on the licenses of the individual physicians and their formal certification in a designated 'Medical Specialty'.

>And with Clinical Trial registry, while the physician organization may be the trial sponsor or site, it is the individual physician who is the Primary Investigator.

## Summary of issues
In 2016 I wrote about the many issues we would face by leaving the `Physican` type in it's current form.  Please review:
Physician person vs Physican organization: https://github.com/schemaorg/schemaorg/issues/280#issuecomment-253915688

>*Insurance Vocab:* The Physician person and the Physician organization are separate entities and have different NPI numbers. Only some physicians in a Physician Organization may participate with an insurance carrier. We must be able to disambiguate the two.

>*Clinical Trial Vocab:* In a Phase I, II, or III Clinical Trial, the Primary Investigator (PI) is a physician person, NOT the organization. But the Clinical Trial Location is typically the Physician organization, and not the person.

>*MedicalSpecialty:* Technically speaking , a Medical Specialty has it's data provenance and lineage traced to the Physician person, and NOT the organization. It is the Physician who attains Board Certification by an authorized Medical Board in a given Medical Specialty. The organization simply inherits the Medical Speciality by virtue of the Physician person(s) who are employed by the organization. The organization can no longer claim a particular Medical Specialty if a physician person of that Medical Specialty leaves the organization. Bottom line, we need to show this foundational relationship of MedicalSpecialty with Physician person.

Hope this helps underscore that we can not merge the physican person and the physician organization into one entity. They are two. CMS disambiguates the difference,  and so should we. 

-Leeza

**Comment by jvandriel** [comment]
I think that would work although having gone through all the (past) issues and initiatives surrounding this topic, made me wonder a few things:

1. #806 also mentions `Dentist` (`DentistOffice`) and `Optician` (OpticiansOffice), which makes me wonder whether these shouldn't get handled in a similar manner? (I have no idea whether dentists and opticians have US NPIs or not)
2. Would it be possible to provide some markup examples for the new classes? 
Reason I ask is because I expect many will want to publish information that will likely end up in folks having to use multi-typed entities (`"@type": ["IndividualPhysician","Person"]` and/or `"@type"["PhysiciansOffice","LocalBusiness"]) based on  the type of information that can often be found on their respective webpages. A few examples that provide some guidance would for such situations would be much appreciated.

**Comment by jvandriel** [comment]
In regards to the markup examples, and just to be clear, I wasn't hinting at any full blown, all encompassing and SEO optimized markup examples. 

I think it's fine to leave that up to any consuming party (like the search engines), but maybe we could at least provide some basic examples illustrating the use of MTEs as many still aren't aware we can make use of them, yet for these particular classes it is to be expected people will make use of them.

Might as well point them the way by providing some basic hints.

**Comment by jvandriel** [comment]
Just to continue a X conversation here... On @danbri wrote: 
_"@rrlevering suggests the PhysiciansOffice type could be a MedicalBusiness. I think this additional change makes sense."_

Which I agree with bu-ut...
The class tree would be:
`Organization > LocalBusiness > MedicalBusiness > PhysiciansOffice`
`Organization > MedicalOrganization > Physician > PhysiciansOffice`

Might it be an idea to also expand the domain of MedicalBusiness to `MedicalOrganization` as well?

Reason being that it's quite cumbersome to have to create multi types entities to be able to express the `medicalSpecialty` of medical businesses or whether they accept new patients.

This change would reflect the same sort of sub typing as `Organization > LocalBusiness` though because `MedicalBusiness` and `MedicalOrganization` live in different branches of the tree we now have to jump through hoops.

**Comment by rrlevering** [comment]
I don't think we were suggesting dropping the MedicalOrganization relationship (most of the predicates in Physician I think need to be inherited because Physician was more a business in schema.org than an individual so I think those things would need to be moved as well).  So I'm suggesting:

[Thing](https://schema.org/Thing) > [Organization](https://schema.org/Organization) > [MedicalOrganization](https://schema.org/MedicalOrganization) > [Physician](https://schema.org/Physician) > [PhysiciansOffice](https://schema.org/PhysiciansOffice)
[Thing](https://schema.org/Thing) > [Organization](https://schema.org/Organization) > [LocalBusiness](https://schema.org/LocalBusiness) > [MedicalBusiness](https://schema.org/MedicalBusiness) > [PhysiciansOffice](https://schema.org/PhysiciansOffice)
[Thing](https://schema.org/Thing) > [Place](https://schema.org/Place) > [LocalBusiness](https://schema.org/LocalBusiness) > [MedicalBusiness](https://schema.org/MedicalBusiness) > [PhysiciansOffice](https://schema.org/PhysiciansOffice)

Note this isn't my favorite option either, but I wouldn't have made the first change like Dan did.

**Comment by jvandriel** [comment]
I agree it's not the most gracious solution but anything else would probably have entailed a much larger overhaul. At least this has a strong use case, while also offering a solution for long standing issues surrounding `Physician`. We can always improve on that further down the road if needed.

**Comment by jvandriel** [comment]
I just realized that the domain of `usNPI` should probably be changed to `MedicalBusiness` because things like `MedicalClinic`s have NPI numbers as well.

Could we please also make that change?

**Comment by Kubot12** [comment]
> ## NPI numbers: Two types (Individual Person, Organization)

I don't see any action for the comments of @LeezaRodriguez on making usNPI usable for organizations too. 

As a use case, we have an NPI for each of our clinics. This is an organization NPI. The doctors may turnover and may have their own private practice NPI.  

We need to be able to use usNPI under the organization, not only under physician.

**Comment by rlzijdeman** [comment]
> in #3427 @jvandriel suggests making Physician a subtype of Occupation to benefit from https://schema.org/occupationalCategory. I am wary of the former as we could end up in a situation like Product where (amongst thousands of possibilities) only Vehicle is modeled as a subtype. However it is a great point that we ought to be able to use the occupationalCategory property, so I suggest we converge that suggestion into the "2 more precise subtypes" design proposed in the PR.

Could you please say what the need is for having 'Physician' to be a subcategory of occupationalCategory. 'Physician' is not a category. In short, 'cashier at McDonalds', 'Physician at my own practice' are sdo:Occupation's where the strings are 'names' or 'alternateNames'. These are coded into occupationalCategories, such as (H)ISCO, SOC and dozens more. These are systems fully representing occupational structures. I have two main concerns with this solution. (1) By only adding 'Physician' you do not add an occupationalCategory. It also goes against my understanding of SDO: occupationalCategory expects a code or text (text here would be 'Physician'). (2) In the past we have reached agreement on the idea that SDO would cover the generics (sdo:Occupation, sdo:OccupationalCategory and sdo:CategoryCode to represent values of sdo:OccupationalCategories. Now you cross that border by adding 1 (or a handful) of occupations as types.
